template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hive/HiveMetastoreCloudStorageServiceConfigProvider.java

```diff
@@ -16,13 +16,17 @@
 @Component
 public class HiveMetastoreCloudStorageServiceConfigProvider implements CmTemplateComponentConfigProvider {
 
+    public static final String HIVE_SERVICE_CONFIG_SAFETY_VALVE = "hive_service_config_safety_valve";
```

Comment:
I don't know for which property this should be added to. On CM UI there is a 
```
Hive Replication Advanced Configuration Snippet (Safety Valve) for hive-site.xml
```
and the corresponding value for that is
```
hive_service_replication_config_safety_valve
```

Reply:
thanks